CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
```

Comment:
The `Salary` property is being added but placed in between attributes and navigation properties. Please follow the convention of keeping scalar properties grouped and navigation properties at the end for clarity.

```suggestion
public decimal Salary { get; set; }
```